<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>538</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/free-icon-ico-5966376.ico</normaloff>../../Downloads/free-icon-ico-5966376.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(84, 180, 199)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxAnswer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(121, 244, 255)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>271</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>BusyCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
border-radius: 10px;
background: rgb(255, 255, 255)</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxTask">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;</string>
     </property>
     <property name="title">
      <string>Выбор задания</string>
     </property>
     <widget class="QRadioButton" name="radioButtonTaskB">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px solid #000000;</string>
      </property>
      <property name="text">
       <string>Б</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonTaskA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px solid #000000;</string>
      </property>
      <property name="text">
       <string>А</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxIFiles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;</string>
     </property>
     <property name="text">
      <string>Запись в файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
border-radius: 20px;
background: rgb(130, 170, 255)</string>
     </property>
     <property name="text">
      <string>Результат</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/icons8-воспроизведение-30.ico</normaloff>../../Downloads/icons8-воспроизведение-30.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditSize">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 1px solid #000000;
background: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Длина массива</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSettings">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(121, 244, 255)</string>
    </property>
    <property name="title">
     <string>Настройки диапазона</string>
    </property>
    <widget class="QRadioButton" name="radioButtonScroll">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;
color:rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Ползунок</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Прокрутка</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;
color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Счетчик</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRanges">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(121, 244, 255)</string>
    </property>
    <property name="title">
     <string>Диапазон</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
border-radius: 20px;
background: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSliders">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(121, 244, 255)</string>
    </property>
    <property name="title">
     <string>Полоса прокрутки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSlider" name="sliderPozitive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;
color:rgb(47, 50, 195)</string>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderNegative">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #000000;
color:rgb(47, 50, 195)</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxScrollBars">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(121, 244, 255)</string>
    </property>
    <property name="title">
     <string>Ползунок</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QScrollBar" name="scrollBarPozitive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(47, 50, 195)</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollBarNigative">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(47, 50, 195)</string>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCircles">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
color: rgb(0, 0, 0);
background: rgb(121, 244, 255)</string>
    </property>
    <property name="title">
     <string>Счетчик</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QDial" name="dialPositive">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(47, 50, 195);
background: rgb(130, 170, 255)</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QDial" name="dialNigative">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(47, 50, 195);
background: rgb(130, 170, 255)</string>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
